libra-node/src/main_node.rs

```diff
@@ -213,14 +211,7 @@ pub fn setup_environment(node_config: &mut NodeConfig) -> LibraHandle {
             .add_mempool(vec![ProtocolId::from_static(MEMPOOL_DIRECT_SEND_PROTOCOL)]);
         mempool_network_handles.push((network.peer_id, mempool_sender, mempool_events));
 
-        let (ac_sender, ac_events) =
-            network_provider.add_admission_control(vec![ProtocolId::from_static(
-                ADMISSION_CONTROL_RPC_PROTOCOL,
```

Comment:
can we also get rid of all references in networking code to  AC rpc protocol

Reply:
I can remove `ADMISSION_CONTROL_RPC_PROTOCOL` refs in this file for this PR - I can delete the actual declaration of it, along with AdmissionControlNetworkEvents and AdmissionControlNetworkSender, in another PR - this PR is getting too big